fjord/suggest/providers/trigger/provider.py

```diff
@@ -37,6 +39,20 @@ def build_ga_category(rule):
     return 'trigger_{rule.slug}_{rule.id}'.format(rule=rule)
 
 
+def interpolate_url(url, response):
+    if '{' not in url:
+        return url
+
+    data = {
+        'LOCALE': urllib.quote_plus(response.locale.encode('utf-8')),
+        'PRODUCT': urllib.quote_plus(response.product.encode('utf-8')),
+        'VERSION': urllib.quote_plus(response.version.encode('utf-8')),
+        'PLATFORM': urllib.quote_plus(response.platform.encode('utf-8')),
+        'HAPPY': 'happy' if response.happy else 'sad',
+    }
+    return url.format(**data)
```

Comment:
url is explicitly set by analyzers in the form. It can't come from anywhere else.

Mmm... I should switch this to a series of `.replace()` calls.